packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -215,6 +243,15 @@ library ChainAssertions {
         require(factory.owner() == _expectedOwner, "CHECK-DG-20");
     }
 
+    /// @notice Asserts that the PreimageOracle is setup correctly
+    function checkPreimageOracle(IPreimageOracle _oracle, DeployConfig _cfg) internal view {
```

Comment:
This is a singleton, and is not currently exposed in the _`proxiesUnstrict()` getter, so unlike other functions in this file I'm just passing in the contract rather than a struct with all of the contracts.

I may add a PR on top of this to clean up those getters a bit.